src/core/propagation/virtual_sites_relative.hpp

```diff
@@ -16,15 +16,14 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-#ifndef VIRTUAL_SITES_VIRTUAL_SITES_OFF_HPP
-#define VIRTUAL_SITES_VIRTUAL_SITES_OFF_HPP
 
-#include "config.hpp"
-#ifdef VIRTUAL_SITES
-#include "VirtualSites.hpp"
+#pragma once
```

Comment:
We cannot use `#pragma once` in ESPResSo because we copy-paste files from `src/` to `build/`. See [Pragma once: Caveats](https://en.wikipedia.org/wiki/Pragma_once#Caveats) for more details.